packages/databricks-vscode/src/configuration/ui/ClusterComponent.ts

```diff
@@ -157,6 +133,14 @@ export class ClusterComponent extends BaseComponent {
     public async getChildren(
         parent?: TreeItem
     ): Promise<ConfigurationTreeItem[]> {
+        // Only show cluster details when we are connected and in dev mode. In other modes
+        // there is no concept of a global interactive cluster
+        if (
+            this.connectionManager.state !== "CONNECTED" ||
+            (await this.configModel.get("mode")) !== "dev"
```

Comment:
Should we show some `Cluster management is only enabled in "dev" mode` child here? Or maybe change the root item to indicate this "non-dev" state?

Same for the sync destination component

Reply:
Created a ticket.